src/chttpd/src/chttpd_view.erl

```diff
@@ -89,16 +95,9 @@ handle_view_req(#httpd{method='POST',
     chttpd:validate_ctype(Req, "application/json"),
     Props = couch_httpd:json_body_obj(Req),
     assert_no_queries_param(couch_mrview_util:get_view_queries(Props)),
-    case couch_mrview_util:get_view_keys(Props) of
-        Keys when is_list(Keys) ->
-            couch_stats:increment_counter([couchdb, httpd, view_reads]),
-            design_doc_view(Req, Db, DDoc, ViewName, Keys);
-        _ ->
-            throw({
-                bad_request,
-                "POST body must contain an array called `keys`"
-            })
-    end;
+    Keys = couch_mrview_util:get_view_keys(Props),
```

Comment:
We should probably remove the debug message from get_view_keys [1] now that we're saying a missing keys field is ok.

[1] https://github.com/apache/couchdb/blob/1fdfe466223380d1ee7c82e21283e161290aed87/src/couch_mrview/src/couch_mrview_util.erl#L1371

Reply:
Fixed in bcbb0ab